pyproject.toml

```diff
@@ -26,6 +26,7 @@ auxtel_housekeeping = "lsst.ts.IntegrationTests.auxtel_housekeeping:run_auxtel_h
 auxtel_image_taking = "lsst.ts.IntegrationTests.image_taking_verification:run_auxtel_image_taking"
 auxtel_latiss_acquire_and_take_sequence = "lsst.ts.IntegrationTests.auxtel_latiss_acquire_and_take_sequence:run_auxtel_latiss_acquire_and_take_sequence"
 auxtel_latiss_calibrations = "lsst.ts.IntegrationTests.auxtel_latiss_calibrations:run_auxtel_latiss_calibrations"
+auxtel_latiss_checkout = "lsst.ts.IntegrationTests.auxtel_latiss_checkout:run_latiss_checkout"
```

Comment:
Your convention for the `run_<blah>` has always been to use what's on the left side of the `=` for `<blah>`. Let's continue using that. The `auxtel_telescope_dome_checkout` doesn't follow this either, so I missed that in the prior review.